client/src/dbus/service.rs

```diff
@@ -51,7 +51,8 @@ impl<'a> Service<'a> {
     pub async fn new() -> Result<Service<'a>, Error> {
         let service = match Self::encrypted().await {
             Ok(service) => Ok(service),
-            Err(Error::Zbus(zbus::Error::MethodError(_, _, _))) => Self::plain().await,
+            Err(Error::ZBus(zbus::Error::MethodError(_, _, _))) => Self::plain().await,
+            Err(Error::Service(ServiceError::ZBus(zbus::Error::MethodError(_, _, _)))) => Self::plain().await,
```

Comment:
```suggestion
            Err(Error::ZBus(zbus::Error::MethodError(_, _, _))) | Err(Error::Service(ServiceError::ZBus(zbus::Error::MethodError(_, _, _)))) => Self::plain().await,
```
maybe? or would that be too unreadable?

Reply:
In this very specific case I would say that it is less readable and the diff is also less readable, considering the rhs is just a single call.

Reply:
I also think it less readable to merge the two already long lines together. I tried to replace `_, _, _` to `..` but the second branch still exceeds the width limit and results in the two branches being formatted differently (the first branch is inline while the second is put into a block). I have updated the pull request with my best effort solution.